tests/org.pitest.pitclipse.ui.tests/src/org/pitest/pitclipse/ui/behaviours/steps/PitclipseSteps.java

```diff
@@ -60,24 +59,24 @@
 public class PitclipseSteps {
 
     @When("test {word} in package {word} is run for project {word}")
-    public void runTest(final String testClassName, final String packageName, final String projectName) throws CoreException {
+    public void runTest(final String testClassName, final String packageName, final String projectName)
+            throws CoreException {
         // No need to do a full build: we should be now synchronized with building
         // Build the whole workspace to prevent random compilation failures
-        // ResourcesPlugin.getWorkspace().build(IncrementalProjectBuilder.FULL_BUILD, new NullProgressMonitor());
-        System.out.println
-            (String.format("Run PIT on: %s %s.%s",
-                projectName, packageName, testClassName));
+        // ResourcesPlugin.getWorkspace().build(IncrementalProjectBuilder.FULL_BUILD,
+        // new NullProgressMonitor());
+        System.out.println(String.format("Run PIT on: %s %s.%s", projectName, packageName, testClassName));
         runPit(new SelectTestClass(testClassName, packageName, projectName));
     }
 
     @Then("a coverage report is generated with {int} class/classes tested with overall coverage of {int}% and mutation coverage of {int}%")
     public void coverageReportGenerated(int classes, double totalCoverage, double mutationCoverage) {
-        PitSummaryView pitView = PAGES.getPitSummaryView();
-        pitView.waitForUpdate();
+        PAGES.views().waitForTestsAreRunOnConsole();
```

Comment:
I guess I should wait before reviewing this right? This should be related to https://github.com/pitest/pitclipse/pull/167

Reply:
Yes, please.